diff -Nur xpdf-2.01.orig/xpdf/XPDFCore.cc xpdf-2.01/xpdf/XPDFCore.cc
--- xpdf-2.01.orig/xpdf/XPDFCore.cc	Sun Nov  3 23:15:37 2002
+++ xpdf-2.01/xpdf/XPDFCore.cc	Mon Nov  4 21:08:51 2002
@@ -906,9 +906,6 @@
 // global variables (currentSelection and currentSelectionOwner).
 
 void XPDFCore::copySelection() {
-  if (!doc->okToCopy()) {
-    return;
-  }
   if (currentSelection) {
     delete currentSelection;
   }
@@ -947,9 +944,6 @@
 }
 
 GString *XPDFCore::extractText(int xMin, int yMin, int xMax, int yMax) {
-  if (!doc->okToCopy()) {
-    return NULL;
-  }
   return out->getText(xMin, yMin, xMax, yMax);
 }
 
@@ -958,9 +952,6 @@
   TextOutputDev *textOut;
   GString *s;
 
-  if (!doc->okToCopy()) {
-    return NULL;
-  }
   textOut = new TextOutputDev(NULL, gFalse, gFalse, gFalse);
   if (!textOut->isOk()) {
     delete textOut;
@@ -1585,10 +1576,9 @@
 #ifndef NO_TEXT_SELECT
 	  if (core->selectXMin != core->selectXMax &&
 	      core->selectYMin != core->selectYMax) {
-	    if (core->doc->okToCopy()) {
-	      core->copySelection();
-	    } else {
-	      error(-1, "Copying of text from this document is not allowed.");
+	    core->copySelection();
+	    if (!core->doc->okToCopy()) {
+	      error(-1, "Note: creator didn't want you to copy text from this document.");
 	    }
 	  }
 #endif
diff -Nur xpdf-2.00.orig/xpdf/XPDFViewer.cc xpdf-2.00/xpdf/XPDFViewer.cc
--- xpdf-2.00.orig/xpdf/XPDFViewer.cc	Sun Nov  3 23:15:37 2002
+++ xpdf-2.00/xpdf/XPDFViewer.cc	Mon Nov  4 21:09:12 2002
@@ -2124,8 +2124,7 @@
 
   doc = viewer->core->getDoc();
   if (!doc->okToPrint()) {
-    error(-1, "Printing this document is not allowed.");
-    return;
+    error(-1, "Note: creator didn't want you to print this document.");
   }
 
   viewer->core->setBusyCursor(gTrue);
diff -Nur xpdf-2.00.orig/xpdf/pdfimages.cc xpdf-2.00/xpdf/pdfimages.cc
--- xpdf-2.00.orig/xpdf/pdfimages.cc	Sun Nov  3 23:15:37 2002
+++ xpdf-2.00/xpdf/pdfimages.cc	Mon Nov  4 21:09:41 2002
@@ -120,9 +120,7 @@
 
   // check for copy permission
   if (!doc->okToCopy()) {
-    error(-1, "Copying of images from this document is not allowed.");
-    exitCode = 3;
-    goto err1;
+    error(-1, "Note: creator didn't want you to copy images from this document.");
   }
 
   // get page range
diff -Nur xpdf-2.00.orig/xpdf/pdftotext.cc xpdf-2.00/xpdf/pdftotext.cc
--- xpdf-2.00.orig/xpdf/pdftotext.cc	Sun Nov  3 23:15:37 2002
+++ xpdf-2.00/xpdf/pdftotext.cc	Mon Nov  4 21:10:18 2002
@@ -153,9 +153,7 @@
 
   // check for copy permission
   if (!doc->okToCopy()) {
-    error(-1, "Copying of text from this document is not allowed.");
-    exitCode = 3;
-    goto err2;
+    error(-1, "Note: creator didn't want you to copy text from this document.");
   }
 
   // construct text file name
