--- xpdf-4.01.01/fofi/CMakeLists.txt.orig	2019-03-14 22:01:02.000000000 +0100
+++ xpdf-4.01.01/fofi/CMakeLists.txt	2019-05-19 10:21:08.075467801 +0200
@@ -24,3 +24,4 @@
 add_library(fofi
   $<TARGET_OBJECTS:fofi_objs>
 )
+target_link_libraries(fofi goo)
--- xpdf-4.01.01/splash/CMakeLists.txt.orig	2019-03-14 22:01:02.000000000 +0100
+++ xpdf-4.01.01/splash/CMakeLists.txt	2019-05-19 10:22:19.315081863 +0200
@@ -44,4 +44,5 @@
   add_library(splash
     $<TARGET_OBJECTS:splash_objs>
   )
+  target_link_libraries(splash fofi goo ${DTYPE_LIBS} ${FREETYPE_LIBS})
 endif ()
