--- xpdf-2.01/xpdf/FontFile.cc.orig	Fri Dec  6 00:44:32 2002
+++ xpdf-2.01/xpdf/FontFile.cc	Thu Dec 26 18:38:36 2002
@@ -18,6 +18,7 @@
 #include <stdarg.h>
 #include <string.h>
 #include <ctype.h>
+#include <locale.h>
 #include "gmem.h"
 #include "Error.h"
 #include "GlobalParams.h"
@@ -358,6 +359,10 @@
   int c, sid;
   int i, j, n;
 
+  /* LC_NUMERIC from environment may cause writing "," as decimal dot
+   * when creating temporary fonts, leading to SEGV when trying to use them */
+  setlocale(LC_NUMERIC, "C");
+
   outputFunc = outputFuncA;
   outputStream = outputStreamA;
 
@@ -614,6 +619,10 @@
   GBool isFP;
   int i, j, k, n;
 
+  /* LC_NUMERIC from environment may cause writing "," as decimal dot
+   * when creating temporary fonts, leading to SEGV when trying to use them */
+  setlocale(LC_NUMERIC, "C");
+
   outputFunc = outputFuncA;
   outputStream = outputStreamA;
 
@@ -901,6 +910,10 @@
   GBool isFP;
   int i, j, n;
 
+  /* LC_NUMERIC from environment may cause writing "," as decimal dot
+   * when creating temporary fonts, leading to SEGV when trying to use them */
+  setlocale(LC_NUMERIC, "C");
+
   outputFunc = outputFuncA;
   outputStream = outputStreamA;
 
@@ -2881,6 +2894,10 @@
 				       void *outputStream) {
   char buf[512];
 
+  /* LC_NUMERIC from environment may cause writing "," as decimal dot
+   * when creating temporary fonts, leading to SEGV when trying to use them */
+  setlocale(LC_NUMERIC, "C");
+
   // write the header
   sprintf(buf, "%%!PS-TrueTypeFont-%g\n", getFixed(0));
   (*outputFunc)(outputStream, buf, strlen(buf));
@@ -2915,6 +2932,10 @@
   Gushort cid;
   int i, j, k;
 
+  /* LC_NUMERIC from environment may cause writing "," as decimal dot
+   * when creating temporary fonts, leading to SEGV when trying to use them */
+  setlocale(LC_NUMERIC, "C");
+
   // write the header
   sprintf(buf, "%%!PS-TrueTypeFont-%g\n", getFixed(0));
   (*outputFunc)(outputStream, buf, strlen(buf));
