--- xpdf-3.01/xpdf/XPDFCore.cc.orig	2005-08-17 07:34:31.000000000 +0200
+++ xpdf-3.01/xpdf/XPDFCore.cc	2005-08-28 11:07:57.279864520 +0200
@@ -377,9 +377,6 @@
   int pg;
   double ulx, uly, lrx, lry;
 
-  if (!doc->okToCopy()) {
-    return;
-  }
   if (getSelection(&pg, &ulx, &uly, &lrx, &lry)) {
     //~ for multithreading: need a mutex here
     if (currentSelection) {
@@ -1108,10 +1105,9 @@
 #ifndef NO_TEXT_SELECT
 	  if (core->selectULX != core->selectLRX &&
 	      core->selectULY != core->selectLRY) {
-	    if (core->doc->okToCopy()) {
-	      core->copySelection();
-	    } else {
-	      error(-1, "Copying of text from this document is not allowed.");
+	    core->copySelection();
+	    if (!core->doc->okToCopy()) {
+	      error(-1, "Note: creator didn't want you to copy text from this document.");
 	    }
 	  }
 #endif
diff -Nur xpdf-2.00.orig/xpdf/XPDFViewer.cc xpdf-2.00/xpdf/XPDFViewer.cc
--- xpdf-2.00.orig/xpdf/XPDFViewer.cc	Sun Nov  3 23:15:37 2002
+++ xpdf-2.00/xpdf/XPDFViewer.cc	Mon Nov  4 21:09:12 2002
@@ -2124,8 +2124,7 @@
 
   doc = viewer->core->getDoc();
   if (!doc->okToPrint()) {
-    error(-1, "Printing this document is not allowed.");
-    return;
+    error(-1, "Note: creator didn't want you to print this document.");
   }
 
   viewer->core->setBusyCursor(gTrue);
diff -Nur xpdf-2.00.orig/xpdf/pdfimages.cc xpdf-2.00/xpdf/pdfimages.cc
--- xpdf-2.00.orig/xpdf/pdfimages.cc	Sun Nov  3 23:15:37 2002
+++ xpdf-2.00/xpdf/pdfimages.cc	Mon Nov  4 21:09:41 2002
@@ -120,9 +120,7 @@
 
   // check for copy permission
   if (!doc->okToCopy()) {
-    error(-1, "Copying of images from this document is not allowed.");
-    exitCode = 3;
-    goto err1;
+    error(-1, "Note: creator didn't want you to copy images from this document.");
   }
 
   // get page range
diff -Nur xpdf-2.00.orig/xpdf/pdftotext.cc xpdf-2.00/xpdf/pdftotext.cc
--- xpdf-2.00.orig/xpdf/pdftotext.cc	Sun Nov  3 23:15:37 2002
+++ xpdf-2.00/xpdf/pdftotext.cc	Mon Nov  4 21:10:18 2002
@@ -153,9 +153,7 @@
 
   // check for copy permission
   if (!doc->okToCopy()) {
-    error(-1, "Copying of text from this document is not allowed.");
-    exitCode = 3;
-    goto err2;
+    error(-1, "Note: creator didn't want you to copy text from this document.");
   }
 
   // construct text file name
--- xpdf-3.01/xpdf/PDFCore.cc.orig	2005-08-17 07:34:31.000000000 +0200
+++ xpdf-3.01/xpdf/PDFCore.cc	2005-08-28 11:08:37.884691648 +0200
@@ -1273,9 +1273,6 @@
   int x0, y0, x1, y1, t;
   GString *s;
 
-  if (!doc->okToCopy()) {
-    return NULL;
-  }
   if ((page = findPage(pg))) {
     cvtUserToDev(pg, xMin, yMin, &x0, &y0);
     cvtUserToDev(pg, xMax, yMax, &x1, &y1);
