diff -ur -x .svn -x .git -x .bzr -x CVS -ur xpdf-4.00/xpdf/pdfimages.cc xpdf-4.00b/xpdf/pdfimages.cc
--- xpdf-4.00/xpdf/pdfimages.cc	2017-08-09 01:22:50.000000000 +0200
+++ xpdf-4.00b/xpdf/pdfimages.cc	2017-11-05 23:15:21.454296468 +0100
@@ -130,7 +130,7 @@
     error(errNotAllowed, -1,
 	  "Copying of images from this document is not allowed.");
     exitCode = 3;
-    goto err1;
+//  goto err1;
   }
 
   // get page range
diff -ur -x .svn -x .git -x .bzr -x CVS -ur xpdf-4.00/xpdf/pdftohtml.cc xpdf-4.00b/xpdf/pdftohtml.cc
--- xpdf-4.00/xpdf/pdftohtml.cc	2017-08-09 01:22:50.000000000 +0200
+++ xpdf-4.00b/xpdf/pdftohtml.cc	2017-11-05 23:15:50.590712980 +0100
@@ -142,7 +142,7 @@
     error(errNotAllowed, -1,
 	  "Copying of text from this document is not allowed.");
     exitCode = 3;
-    goto err1;
+//  goto err1;
   }
 
   // get page range
diff -ur -x .svn -x .git -x .bzr -x CVS -ur xpdf-4.00/xpdf/pdftops.cc xpdf-4.00b/xpdf/pdftops.cc
--- xpdf-4.00/xpdf/pdftops.cc	2017-08-09 01:22:50.000000000 +0200
+++ xpdf-4.00b/xpdf/pdftops.cc	2017-11-05 23:18:25.649219651 +0100
@@ -325,7 +325,7 @@
   if (!doc->okToPrint()) {
     error(errNotAllowed, -1, "Printing this document is not allowed.");
     exitCode = 3;
-    goto err1;
+//  goto err1;
   }
 
   // construct PostScript file name
diff -ur -x .svn -x .git -x .bzr -x CVS -ur xpdf-4.00/xpdf/pdftotext.cc xpdf-4.00b/xpdf/pdftotext.cc
--- xpdf-4.00/xpdf/pdftotext.cc	2017-08-09 01:22:50.000000000 +0200
+++ xpdf-4.00b/xpdf/pdftotext.cc	2017-11-05 23:16:08.077224470 +0100
@@ -203,7 +203,7 @@
     error(errNotAllowed, -1,
 	  "Copying of text from this document is not allowed.");
     exitCode = 3;
-    goto err2;
+//  goto err2;
   }
 
   // construct text file name
diff -ur -x .svn -x .git -x .bzr -x CVS -ur xpdf-4.00/xpdf-qt/QtPDFCore.cc xpdf-4.00b/xpdf-qt/QtPDFCore.cc
--- xpdf-4.00/xpdf-qt/QtPDFCore.cc	2017-08-09 01:22:50.000000000 +0200
+++ xpdf-4.00b/xpdf-qt/QtPDFCore.cc	2017-11-05 23:17:00.560070752 +0100
@@ -430,7 +430,7 @@
   int i;
 
   if (!doc->okToCopy()) {
-    return "";
+//  return "";
   }
   if (!(s = getSelectedText())) {
     return "";
@@ -459,7 +459,7 @@
     return;
   }
   if (!doc->okToCopy()) {
-    return;
+//  return;
   }
   if (hasSelection()) {
     QApplication::clipboard()->setText(getSelectedTextQString(),
